packages/connect/src/api/firmware/__tests__/verifyAuthenticityProof.test.ts

```diff
@@ -12,25 +12,27 @@ const CHALLENGE = '29d0be0f3cb191c80d108359c64d22984a77ad8b99433814be31db0b6e9e7
 const CONFIG = {
```

Comment:
```suggestion
const CONFIG: DeviceAuthenticityConfig = {
```

instead of `as` cast the missing `T3B1` would then scream as TS error long before this change

Reply:
good catch